server/models/node.js

```diff
@@ -66,40 +89,37 @@ async function getList(chain_id, from, num) {
     });
 }
 
-async function getHistory(chain_id, node_id, anchor, from, num) {
+async function getHistory(chain_id, val_addr, anchor, from, num) {
```

Comment:
What is the difference between this and `getOne` ? almost the same signature except `anchor`.

Reply:
- `getOne`: latest status of a node given from client (single value)
- `getList`: latest status of nodes (list)
- `getHistory`: history status of a node given from client (list)